### PR TITLE
Add sql migration scripts

### PR DESCRIPTION
**Pull request by FirestarJes**
Add migration scrips for all environments

### CODE REVIEW

File: source/Energinet.DataHub.ActorRegistry.ApplyDBMigrationsApp/Helpers/EnvironmentFilter.cs [hunk 1]
```diff
@@ -13,18 +13,37 @@
...
 
 using System;
+using System.Collections.Generic;
 using System.Linq;
 
 namespace Energinet.DataHub.ActorRegistry.ApplyDBMigrationsApp.Helpers
 {
     public static class EnvironmentFilter
     {
+        private static readonly string[] _validEnvironments = { "u-001", "u-002", "t-001", "t-002", "b-001", "b-002", "U-001", "U-002", "T-001", "T-002", "B-001", "B-002" };
```
Comment (djorgensendk) [inline source/Energinet.DataHub.ActorRegistry.ApplyDBMigrationsApp/Helpers/EnvironmentFilter.cs:23]: we don't have a t-002 env (nor T-002)
Why are the environment letters both in lower and capital?
Resources in azure are in lower-case - except resource groups where camel casing is used

File: source/Energinet.DataHub.ActorRegistry.ApplyDBMigrationsApp/Scripts/B-001/Seed/202201311101 Seed ActorInfo Table.sql [hunk 2]
```diff
@@ -0,0 +1,67 @@
+INSERT [dbo].[ActorInfo] ([Id], [IdentificationNumber], [IdentificationType], [Roles], [Active], [Name]) VALUES (N'3000e8c2-0dc0-4532-a40d-01ca17fe0d08', N'5790000681327', 1, N'DDM,MDR', 1, N'RAH Net A/S')
```
Comment (djorgensendk) [inline source/Energinet.DataHub.ActorRegistry.ApplyDBMigrationsApp/Scripts/B-001/Seed/202201311101 Seed ActorInfo Table.sql:1]: there are no app registrations/organizations in the B-001 B2C

**Review by djorgensendk — CHANGES_REQUESTED** [review]

File: source/Energinet.DataHub.ActorRegistry.ApplyDBMigrationsApp/Helpers/EnvironmentFilter.cs (hunk 1 above)
Comment (FirestarJes) [reply source/Energinet.DataHub.ActorRegistry.ApplyDBMigrationsApp/Helpers/EnvironmentFilter.cs]: Fixed

File: source/Energinet.DataHub.ActorRegistry.ApplyDBMigrationsApp/Scripts/B-001/Seed/202201311101 Seed ActorInfo Table.sql (hunk 2 above)
Comment (FirestarJes) [reply source/Energinet.DataHub.ActorRegistry.ApplyDBMigrationsApp/Scripts/B-001/Seed/202201311101 Seed ActorInfo Table.sql:1]: It's accepted, we will add new scripts later to cleanup the data.

**Review by djorgensendk — APPROVED** [review]

**Pull request merged**